src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: medium;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+.home {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px;
+  margin-left: 50px;
+}
+
+.nav-bar {
+  margin-right: 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+}
+
+.nav-item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav-link {
+  color: #000;
+  line-height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  cursor: pointer;
+  display: block;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  background-color: #00acdc;
+  width: 37px;
```

Comment:
Why have you hardcoded this width? Other links have different lengths, so it's not works that way